### ISSUE TITLE
Feature request: "[ home ]" link in the gallery.

### ISSUE THREAD

**Issue by van-de-bugger** [chrissimpkins/codeface#175]
Each font in the gallery is represented as:

> **FontName**
> 
> [ License ]
> 
> (specimen images follows here...)

It would be great to see "[ Home ]" link next (or before) the "[ License ]" link, e. g.:

> **FontName**
> 
> [ Home ] [ License ]
> 
> (specimen images follows here...)

**Comment by chrissimpkins** [comment]
And this links to top of README?

**Comment by van-de-bugger** [comment]
> And this links to top of README?

No. I would like it links to the font's home page. Information about font's home for many fonts is present in the repo's `source.txt` files (e. g. `codeface/fonts/agave/source.txt`, `codeface/fonts/average-mono/source.txt`, etc).
